docs/misc/changelog.rst

```diff
@@ -19,13 +19,17 @@ New Features:
   automatic check for image spaces.
 - ``VecFrameStack`` now has a ``channels_order`` argument to tell if observations should be stacked
   on the first or last observation dimension (originally always stacked on last).
+- Add ``DictRolloutBuffer`` to support dictionary observations (need to be tested in ReplayBuffer)
```

Comment:
please don't forget to add your username (and at the bottom too ;))
entries without usernames are from maintainers.